install_mac.sh

```diff
@@ -16,20 +16,33 @@ brew install cmake
 # Install LLVM and clang stuff
 xcode-select install
 if [[ -x "$(command -v rustup)" ]]; then
-  echo "LLVM detected"
+	echo "LLVM detected"
 else
-  brew install llvm
-  echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >>~/.zshrc
+	brew install llvm
+	echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.zshrc
 fi
 
 # Install Rust
 if [[ -x "$(command -v rustup)" ]]; then
-  rustup update
+	rustup update
 else
-  curl https://sh.rustup.rs -sSf | bash -s -- -y
+	curl https://sh.rustup.rs -sSf | bash -s -- -y
 fi
 source "$HOME/.cargo/env"
 export PATH="$HOME/.cargo/bin:$PATH"
 
 # Build project
 cargo build --release
+
+# Install WASI-SDK 1 2to /opt/wasi-sdk, the path wasi-sdk assumed by default
```

Comment:
I pushed some additional changes. The Makefiles and Python bench script now detect the installation location based on the WASI_SDK_PATH environment variable as shown in the WASI-SDK readme. The install script for Mac now uses the path format used by brew for llvm (/usr/local/opt/wasi-sdk). However, you should now be able to theoretically setup wasi-sdk however you want now as long as you set WASI_SDK_PATH in your environment. I still would consider this a "global installation" (since the environment variable is global), but I think this seems to be the happy path for this tooling (it's the pattern used in the README and the dpkg that the maintainers public). Installing in this way also allows us to replace relative paths with environment variables, which is helpful for any follow-on efforts to move the native -> *.wasm compilation stuff into a shared submodule.